browser/src/UI/components/CommandLine.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import { connect } from "react-redux"
+import styled, { keyframes } from "styled-components"
+
+// import * as UI from "./../../UI"
+import * as State from "./../State"
+
+const CommandLineBackground = styled.div`
+    position: absolute;
+    top: 0px;
+    left: 0px;
+    bottom: 0px;
+    right: 0px;
+    background-color: rgba(0, 0, 0, 0.25);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`
+
+const fadeInAndDown = keyframes`
+    from {
+        opacity: 0;
+    }
+    to {
+        opacity: 1;
+    }
+`
+
+const CommandLineBox = styled.div`
+    position: relative;
+    margin-top: 16px;
+    padding: 8px;
+    width: 75%;
+    max-width: 900px;
+    background-color: ${p => p.theme.background};
```

Comment:
Cool, seems like this what you were mentioning - I need to try this out in #1159 !